SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;
+
+Use ZooManagement;
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species VARCHAR(150),
+SpeciesNumber INT,
+AnimalsNumber INT,
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Day VARCHAR(150),
+NumberOfVisitors INT,

```

Comment:
The visitors and the tickets should be in a one to many relationship which can be achieved by using foreign keys. A visitor can purchase multiple tickets and a ticket ca be purchased by a single visitor.